docs/ppp/pitch-23112022.md

```diff
@@ -0,0 +1,91 @@
+## The project
+
+**10101** (speaks, **Ten** **Ten** **One**). One App, All Things Bitcoin!
+
+10101 is a non-custodial Lightning wallet infused with financial services such as CFD trading, sports betting and savings.
+
+## The team
+
+We are a group of 5 developers who initially built [ItchySats](https://itchysats.network/) together. Now we are bringing non-custodial financial services to Lightning onto your phone.
+
+- bonomat, aka Philipp
+
+  - [https://github.com/bonomat](https://github.com/bonomat)
+  - [https://makers.bolt.fun/profile/363/bonomat](https://makers.bolt.fun/profile/363/bonomat)
+
+- da-kami, aka Daniel
+
+  - [https://github.com/da-kami](https://github.com/da-kami)
+  - [https://makers.bolt.fun/profile/381/daniel](https://makers.bolt.fun/profile/381/daniel)
+
+- holzeis, aka Richard
+
+  - [https://github.com/holzeis](https://github.com/holzeis)
+  - [https://makers.bolt.fun/profile/408/holzeis](https://makers.bolt.fun/profile/408/holzeis)
+
+- klochowicz, aka Mariusz
+
+  - [https://github.com/klochowicz](https://github.com/klochowicz)
+  - [https://makers.bolt.fun/profile/461/mariusz](https://makers.bolt.fun/profile/461/mariusz)
+
+- luckysori, aka Lucas
+
+  - [https://github.com/luckysori](https://github.com/luckysori)
+  - [https://makers.bolt.fun/profile/534/luckysori](https://makers.bolt.fun/profile/534/luckysori)
+
+## The next steps
+
+![](https://imagedelivery.net/wyrwp3c-j0gDDUWgnE7lig/166aa91f-7136-4f4e-e35e-d4b1a7eefc00/public)
+
+1. Evaluate
+
+   1. With the outcome of the tournament we want to collect feedback from users to learn how to build the right product
```

Comment:
```suggestion
   1. With the outcome of the tournament we will to collect feedback from users to learn more about the users and the product needs
```

Note: I feel this sounds like we are not sure who we are targeting. Maybe change it to:

```
1. With the outcome of the tournament we want to collect more user feedback to learn more about our target user group and their product needs
```

Our target user group is changing with 10101, I think this should come out when we write it.